deploy/ansible/worker/include/run_external_geth.yml

```
@@ -0,0 +1,22 @@
+- name: "Run shared externally available geth node"
+  hosts: "{{ play_hosts }}"
+  remote_user: "{{default_user}}"
+  gather_facts: no
+  tasks:
+
+    - name: "run geth {{geth_options}} forever in the background"
+      become: yes
+      docker_container:
+        name: geth
+        state: started
+        restart: yes
+        image: ethereum/client-go:latest
```

```suggestion
        image: ethereum/client-go:stable
```
For some reason, their defaults point to development versions, which is an odd choice. We've had some problems in the past trying to debug errors because of this, as the version we sometimes ran was none of the formal releases. I think the `stable` option is a safer choice, although I'm not sure what's their criteria for label as "stable".

---

good idea!